docs/metrics/README.md

```diff
@@ -356,17 +356,19 @@ predictable and reliable behavior when excessive cardinality happens, whether it
 was due to a malicious attack or developer making mistakes while writing code.
 
 OpenTelemetry has a default cardinality limit of `2000` per metric. This limit
-can be configured at `MeterProvider` level using the
-`SetMaxMetricPointsPerMetricStream` method, or at individual
-[view](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view)
-level using `MetricStreamConfiguration.CardinalityLimit`. Refer to this
-[doc](../../docs/metrics/customizing-the-sdk/README.md#changing-maximum-metricpoints-per-metricstream)
+can be configured at the individual metric level using the [View
+API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view)
+and the `MetricStreamConfiguration.CardinalityLimit` setting. Refer to this
+[doc](../../docs/metrics/customizing-the-sdk/README.md#changing-the-cardinality-limit-for-a-metric)
 for more information.
 
 Given a metric, once the cardinality limit is reached, any new measurement which
-cannot be independently aggregated because of the limit will be aggregated using
+cannot be independently aggregated because of the limit will be dropped (a
+warning is written to the [self-diagnostic
```

Comment:
Want to double check if we actually log warning here.. I think we warn only the first time the limit is hit, not for every measurement.. (This is critical, as a user who starts self-diag moments after the point cardinality limit is reached will see no logs about measurements being dropped. but if they had self diag from the beginning, log will be found.)

Reply:
You are right we don't spam the log, only on the first overflow: https://github.com/open-telemetry/opentelemetry-dotnet/blob/f214d27e9331426854836a266cb2d57c75333ed1/src/OpenTelemetry/Metrics/AggregatorStore.cs#L940-L943

I updated the doc to clarify the behavior: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5328/commits/8a6d2c2f4c73e5fc0b273f2dfc203d4b31902bc5